chapters/interfaces.adoc

```diff
@@ -2249,11 +2249,13 @@ culled, if it is code:false it will not be culled.
     The variable decorated with code:CullPrimitiveEXT must: be declared
     using the code:Output {StorageClass}
   * [[VUID-{refpage}-CullPrimitiveEXT-07036]]
-    The variable decorated with code:CullPrimitiveEXT must: be declared as
-    an array of boolean values
-  * [[VUID-{refpage}-CullPrimitiveEXT-07037]]
-    The size of the array decorated with code:CullPrimitiveEXT must: match
-    the value specified by code:OutputPrimitivesEXT
+    code:CullPrimitiveEXT must: be declared as a boolean value member of a structure or a variable of type code:OpTypeArray of boolean values. 
+  * If code:CullPrimitiveEXT declared as an array of boolean values. The size of the array 
+    decorated with code:CullPrimitiveEXT must: match the value specified by code:OutputPrimitivesEXT
+  * If code:CullPrimitiveEXT is a member of a structure, there must: be a variable declared 
+    which is array of parent strucure that contains code:CullPrimitiveEXT. The size of the array must: match the value specified by code:OutputPrimitivesEXT.
+  * There must be only one declaration of the code:CullPrimitiveEXT associated with a entry
+    point's interface.
```

Comment:
```suggestion
    code:CullPrimitiveEXT must: decorate a scalar boolean member of a structure decorated as code:Block, or decorate a variable of type code:OpTypeArray of boolean values. 
  * If code:CullPrimitiveEXT is declared as an array of boolean values, the size of the array 
    must: match the value specified by code:OutputPrimitivesEXT
  * If code:CullPrimitiveEXT decorates a member of a structure, the variable declaration of the containing code:Block type must: have an array size that matches the value specified by code:OutputPrimitivesEXT
  * There must be only one declaration of the code:CullPrimitiveEXT associated with a entry
    point's interface.
```

Reply:
I think this language reads closer to the rest of the spec, but otherwise, the gist of it is fine.

Reply:
Thanks for the suggestion, I am now adding same langugage for rest of the builtins.